graylog2-plugin-interfaces/pom.xml

```diff
@@ -106,6 +106,12 @@
             <groupId>com.github.joschi</groupId>
             <artifactId>jadconfig</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>javax.validation</groupId>
+            <artifactId>validation-api</artifactId>
+            <version>1.1.0.Final</version>
```

Comment:
Please add a corresponding entry into the `dependencyManagement` section of `graylog2-parent`, so that the dependency is being pulled in using the same version everywhere.